CHANGELOG.md

```diff
@@ -22,6 +22,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Removed all Google AMP template files [#2308](https://github.com/bigcommerce/cornerstone/pull/2308)
 - Webpack 5, Node 18 Support [#2311][https://github.com/bigcommerce/cornerstone/pull/2311]
 - Customer order summary with both physical and digital items shows shipping as null [#2309](https://github.com/bigcommerce/cornerstone/pull/2309)
+- Bump Stencil utils to 6.15.0 [#2327][https://github.com/bigcommerce/cornerstone/pull/2327]
```

Comment:
@bc-rmalyavc could you please move this marker to `## Draft` as this change will be included in the next release version

Reply:
Sure, @bc-yevhenii-buliuk. Done. Thanks for a hint